config.sample.yml

```diff
@@ -146,6 +146,10 @@ widgets:
   publicUrl: http://example.com/widgetapi/v1/static
   branding:
     widgetTitle: Hookshot Configuration
+  openIdOverrides:
+    # For testing only: A set of homeserver servernames mapped to their Server-Server API endpoints. This can be used if your local homeserver doesn't lookup via SRV or .well-known.
+    #
+    my-local-server: {}
```

Comment:
The empty object here doesn't match what Config/Defaults seems to set. What's the format here, and what should one put here for it to work? Does `my-local-server` specifically have any meaning?

Reply:
I still don't know what the structure is supposed to be :/ Locally I did a 

```
openIdOverrides:
    home.tadzik.net: "http://home.tadzik.net:8080"
```

And that seems to work, but that's not what the documented structure is – it is a `string` => `object` mapping in the docs, while `string` => `string` is what the code expects (and what seems to work).